frontend/src/MuiForms/TagSelectorBeta.tsx

```diff
@@ -0,0 +1,78 @@
+import { Autocomplete, Box, Chip, Stack, TextField, Typography } from '@mui/material'
+import { useTheme } from '@mui/material/styles'
+
+export default function TagSelector(props: any) {
+  const { onChange, value: currentValue, label, formContext } = props
```

Comment:
Seems unnecessary to destructure `props` like this when you can just do it on the line above as we usually do.